server/channels/api4/channel.go

```diff
@@ -697,16 +708,19 @@ func getChannelsMemberCount(c *Context, w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	channelIDs := model.ArrayFromJSON(r.Body)
-	if !c.App.SessionHasPermissionToChannels(c.AppContext, *c.AppContext.Session(), channelIDs, model.PermissionReadChannel) {
+	channelIds, err := model.ArrayFromJSONLimited(r.Body, *c.App.Config().ServiceSettings.MaximumPayloadSizeBytes)
```

Comment:
Should we check if `channelIds` is empty?

Reply:
Understood. I'd expect a sane client not to send empty arrays but maybe we can defer this and plan a proper review since it may take more time to check everything and there are no reasons to delay the other improvements.